### PR TITLE
Sales

### PR DESCRIPTION
**Pull request by DJCarstens**
Sets up backend functionality for ordered items view

### CODE REVIEW

**Review by keanutex — CHANGES_REQUESTED** [review]
The insert sql code will not work for the products table if images are changed to varbinary. Change the default insert query before merging.

**Review by keanutex — CHANGES_REQUESTED** [review]
This will break the product table sql inserts for the db.

File: Database/CreateProductTable.sql [hunk 1]
```diff
@@ -7,6 +7,6 @@ CREATE TABLE CoroNacessitiesDB.dbo.Product(
 [Price] DECIMAL(10,2) NOT NULL,
 [ProductStatusID] INT NOT NULL,
 [TypeID] INT NOT NULL,
-[ProductImage] VARCHAR(MAX),
+[ProductImage] VARBINARY(MAX),
```
Comment (keanutex) [inline Database/CreateProductTable.sql:10]: This breaks the product table sql insert queries.

Comment (keanutex) [reply Database/CreateProductTable.sql:10]: Type mismatch now.

Comment (DJCarstens) [reply Database/CreateProductTable.sql:10]: Is it actually giving errors or are the data entries just not succeeding?

Comment (keanutex) [reply Database/CreateProductTable.sql:10]: It gives errors if you run it in micosoft sql studio, but not if you run it in console. In console it looks like it just skips over them.

Comment (keanutex) [reply Database/CreateProductTable.sql:10]: You can put whatever temp values in there for now, just as long as they work.

Comment (DJCarstens) [reply Database/CreateProductTable.sql:10]: So would you prefer I change this back to a string or that I change the data entries?

Comment (keanutex) [reply Database/CreateProductTable.sql:10]: Change the data entries. The backend is expecting varbinary.

Comment (DJCarstens) [reply Database/CreateProductTable.sql:10]: Trying to figure out how to store images as blobs now

**Review by keanutex — APPROVED** [review]

**Pull request merged**